src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -246,20 +247,26 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         vs.Insert(L"commandline", Windows::Foundation::PropertyValue::CreateString(cmdline));
         vs.Insert(L"startingDirectory", Windows::Foundation::PropertyValue::CreateString(startingDirectory));
         vs.Insert(L"startingTitle", Windows::Foundation::PropertyValue::CreateString(startingTitle));
+        vs.Insert(L"reloadEnvironmentVariables", Windows::Foundation::PropertyValue::CreateBoolean(reloadEnvironmentVariables));
```

Comment:
I can't figure out where this guy is used. The ValueSet is an internal contract between ConptyConnection and ConptyConnection, so I expected to see it in ConptyConnection.

Reply:
It's in `ConptyConnection::Initialize`. That isn't part of the diff because it was already there, I'm just moving the setting into `ConptyConnection`. Setting it outside, in TerminalPage, is wack.

Reply:
good call thx